src/main/java/com/iri/training/repository/ImageRepositoryImpl.java

```diff
@@ -70,6 +72,7 @@ public Image getProfileImage(Long userId) throws SQLException {
 		jdbcTemplate = new JdbcTemplate(dataSource);
 		final List<Image> images = jdbcTemplate.query(sql, new Object[]{userId}, new ImageListResultSetExtractor());
 
+		logger.debug("RETRIEVED " + images.size() + " images");
 		logger.debug("EXITING getUserImages");
```

Comment:
Print all images objects found in the list (see some solutions here https://stackoverflow.com/questions/10168066/how-to-print-out-all-the-elements-of-a-list-in-java - please pick a one-liner :) )